cmd/security-agent/common/check_command.go

```diff
@@ -125,6 +136,25 @@ func runCheck(cmd *cobra.Command, confPath *string, args []string) error {
 	return nil
 }
 
+func configureLogger() error {
+	var (
+		logFormat = "%LEVEL | %Msg%n"
+		logLevel  = "info"
+	)
+	if checkArgs.verbose {
+		const logDateFormat = "2006-01-02 15:04:05 MST"
+		logFormat = fmt.Sprintf("%%Date(%s) | %%LEVEL | (%%ShortFilePath:%%Line in %%FuncShort) | %%Msg%%n", logDateFormat)
```

Comment:
Should you re-use log format defined in `util/log`?

Reply:
The subtle difference is the lack of the logger name here.